### DISCUSSION TITLE
[4.x]: Multisite ordering of structured entries per site

### DISCUSSION THREAD

**Discussion by iamtompickering** [craftcms/cms#13872] [Ideas]
### Description
I was wondering if it is possible to reorder entries in a structure section manually per site in the CMS. 

I don't really want to go down the route of setting up an entries field where you can add all the entries and reorder them from there as there are 100+ entries, so adding all them to an entries field isn't going to be very performant. 

Is there anything in place currently in Craft which allows me to get around this?

### Craft CMS version

4.5.6.1

### PHP version

8.0

### Installed plugins and versions

-

**Comment by brandonkelly** [comment]
This isn’t currently possible, no, so I’ve converted your issue to a discussion.
